<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.27">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Human Genetics – Pheno.AI</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.min.js" integrity="sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2" crossorigin="anonymous"></script><script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-ed96de9b727972fe78a7b5d16c58bf87.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark-4d9afe2b8d18ee9fa5d0d57b5ed4214d.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-ed96de9b727972fe78a7b5d16c58bf87.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-bd46602fe90269a447d98623afced56a.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark-aeaf3e155c33cdd5728541c7b417a725.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="../site_libs/bootstrap/bootstrap-bd46602fe90269a447d98623afced56a.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="../site_libs/quarto-diagram/mermaid.min.js"></script>
<script src="../site_libs/quarto-diagram/mermaid-init.js"></script>
<link href="../site_libs/quarto-diagram/mermaid.css" rel="stylesheet">
<script src="https://cdn.jsdelivr.net/npm/requirejs@2.3.6/require.min.js" integrity="sha384-c9c+LnTbwQ3aujuU7ULEPVvgLs+Fn6fJUvIGTsuu1ZcCf11fiEubah0ttpca4ntM sha384-6V1/AdqZRWk1KAlWbKBlGhN7VG4iE/yAZcO6NZPMF8od0vukrvr0tg4qY6NSrItx" crossorigin="anonymous"></script>

<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="../styles.css">
</head>

<body class="nav-sidebar docked nav-fixed quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Pheno.AI</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../participant_journey.html"> 
<span class="menu-text">Get to Know the HPP</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../publications.html"> 
<span class="menu-text">Publications</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../datasets_description.html">Datasets</a></li><li class="breadcrumb-item"><a href="../category_multi_omics.html">Multi-Omics</a></li><li class="breadcrumb-item"><a href="../datasets/014-human_genetics.html">Human Genetics</a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Datasets</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets_description.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Datasets Overview</span></a>
  </div>
</li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_study_core.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Study Core</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/000-population.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Population</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/001-events.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Events</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_clinical_assessments.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Clinical Assessments</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/002-anthropometrics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Anthropometrics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/006-hand_grip.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Hand Grip Strength</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/007-blood_pressure.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Blood Pressure</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/010-peripheral_vascular_health.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Peripheral Vascular Health</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/015-ecg.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">ECG (Electrocardiogram)</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/019-cbc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Complete Blood Count (CBC)</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/027-voice.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Voice Recording</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/032-gait.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gait and Balance</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_medical_imaging.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Medical Imaging</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/003-fundus.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Fundus Photography</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/004-liver_ultrasound.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Liver Ultrasound</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/011-body_composition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Body Composition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/012-carotid_ultrasound.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Carotid Ultrasound</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/022-bone_density.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Bone Density</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_multi_omics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Multi-Omics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/008-serum_lipidomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Serum Lipidomics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/013-gut_microbiome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gut Microbiome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/014-human_genetics.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">Human Genetics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/024-rna_seq.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">RNA Sequencing</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/028-olink_proteomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Olink Proteomics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/029-nightingale_metabolomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Nightingale Metabolomics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/030-vaginal_microbiome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Vaginal Microbiome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/034-untargeted_metabolomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Untargeted Metabolomics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/070-oral_microbiome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Oral Microbiome</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_sensors_app_logging.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Sensors and App Logging</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/005-diet_logging.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Diet Logging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/009-sleep.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Sleep</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/017-cgm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CGM (Continuous Glucose Monitoring)</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/033-google_fit_apple_health.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wearables</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_medical_records_surveys.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Medical Records and Surveys</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/016-blood_tests.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Blood Tests</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/018-medications.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Medications</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/021-medical_conditions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Medical Conditions</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/043-covid.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">COVID-19</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/044-sex_specific_factors.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Sex-Specific Factors</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/045-participant_relation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Participant Relations</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/049-medical_symptoms.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Medical Symptoms</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/050-medical_procedures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Medical Procedures</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/051-children.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Children</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/052-family_history.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Family History</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/053-sociodemographics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Sociodemographics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/055-lifestyle_and_environment.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Lifestyle and Environment</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/057-psychological_and_social_health.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Psychological and Social Health</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/058-health_and_medical_history.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Health and Medical History</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../category_derived_phenotypes.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Derived Phenotypes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/100-curated_phenotypes.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Curated Phenotypes</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#description" id="toc-description" class="nav-link active" data-scroll-target="#description">Description</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#measurement-protocol" id="toc-measurement-protocol" class="nav-link" data-scroll-target="#measurement-protocol">Measurement protocol</a></li>
  <li><a href="#data-availability" id="toc-data-availability" class="nav-link" data-scroll-target="#data-availability">Data availability</a></li>
  <li><a href="#summary-of-available-data" id="toc-summary-of-available-data" class="nav-link" data-scroll-target="#summary-of-available-data">Summary of available data</a></li>
  <li><a href="#relevant-links" id="toc-relevant-links" class="nav-link" data-scroll-target="#relevant-links">Relevant links</a></li>
  <li><a href="#data-dictionary" id="toc-data-dictionary" class="nav-link" data-scroll-target="#data-dictionary">Data dictionary</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">


<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../datasets_description.html">Datasets</a></li><li class="breadcrumb-item"><a href="../category_multi_omics.html">Multi-Omics</a></li><li class="breadcrumb-item"><a href="../datasets/014-human_genetics.html">Human Genetics</a></li></ol></nav>
<div class="quarto-title">
<h1 class="title">Human Genetics</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="description" class="level3">
<h3 class="anchored" data-anchor-id="description">Description</h3>
<p>The Human Phenotype Project collects genomic variation data on all its participants. The genomic data together with the Human Phenotype Project deep-phenotypes allows to investigate the progression of disease, and to explore personalized treatments. We genotype millions of positions by low-pass sequencing combined with imputation using gencove platform technologies. Genotype imputation is a process of statistically inferring unobserved genotypes using known haplotypes in a population. The performance of Gencove genotype imputation is very high ( accuracy &gt; 98% ) (<a href="https://link.springer.com/article/10.1186/s12864-021-07508-2?utm_campaign=2022.01%20Publications&amp;utm_source=LP%20for%20Pharmacogenomics">Wasik et al.&nbsp;2021</a>).</p>
</section>
<section id="introduction" class="level3">
<h3 class="anchored" data-anchor-id="introduction">Introduction</h3>
<p>Genomic variation refers to DNA sequence differences between individuals. Some genetic variants can influence biological function (such as a mutation that causes a genetic disease), while others have no known effects. The genomic data together with the Human Phenotype Project rich variety of phenotypic data enables investigation of how genetics impact the progression of disease, and exploration of personalized prevention strategies and treatments.</p>
<p>The Human Phenotype Project uses low-pass sequencing combined with <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">imputation</a> to obtain genomic variation, using <a href="https://gencove.com">genocve</a> platform technologies. We do not target specific regions by sequencing, but rather sample the entire genome with low average depth of coverage, and impute the sequence data to the haplotype reference panel which includes a large set of known genetic variants. Briefly, the model approach is to identify haplotypes shared between a study sample and a reference panel based on the study sample observed genotypes. Next, the unobserved genotypes are extrapolated according to the large reference data as illustrated in Figure 1. Imputation is performed for single nucleotide polymorphisms (SNPs) and short insertion–deletion variants (indels), structural variation are currently not included. The overall accuracy of gencove imputed genotypes is 98.22% (Wasik et al.&nbsp;2021). We also validated the accuracy of the imputation using two hundred Human Phenotype Project participants from several ancestries which had both low-pass sequencing done and Illumina genotyping array data. We obtained high concordance rates, greater than 96%.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="014-human_genetics/study_sample.png" class="img-fluid figure-img"></p>
<figcaption>study sample</figcaption>
</figure>
</div>
<p>Figure 1: taken from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2925172/">Li et al.&nbsp;2013</a>. Panel A illustrates the observed data which consists of genotypes at a modest number of genetic markers in each sample being studied and of detailed information on genotypes (or haplotypes) for a reference sample. Panel B illustrates the process of identifying regions of chromosomes shared between a study sample and individuals in the reference panel. In Panel C, observed genotypes and haplotype sharing information have been combined to fill in a series of unobserved genotypes in the study sample.*</p>
</section>
<section id="measurement-protocol" class="level3">
<h3 class="anchored" data-anchor-id="measurement-protocol">Measurement protocol</h3>
<!-- long measurment protocol for the data browser -->
<section id="genotyping-procedure" class="level4">
<h4 class="anchored" data-anchor-id="genotyping-procedure">Genotyping Procedure</h4>
<p>DNA is collected from Human Phenotype Project participants using a buccal swab during their baseline visit. DNA is collected only once as germline DNA is considered constant. The buccal swabs are shipped to Gencove laboratories. The entire sequencing and genotyping workflow is performed by Gencove technologies. The Detailed information about the Gencove algorithms and quality control protocols are available in Wasik et al.&nbsp;2021, we briefly describe the key steps here.</p>
<p>Gencove’s workflow contains three steps: DNA extraction and library preparation, low-pass whole genome sequencing, and imputation analysis. Sequencing is performed at Neogen GeneSeek Laboratories using Illumina Novaseq at low-pass, targeting coverage levels of 0.5× and 1×. Whole-genome coverage is defined as the number of mapped DNA bases divided by the size of the genome (∼3 billion base pairs). We perform paired-end sequencing at length of 150-bp resulting in two FASTQ files (R1 and R2). Sequencing at paired-end mode significantly improves alignment to the genome. Next, the FASTQ files are aligned to the hs37-1kg reference genome, resulting in a BAM (Binary Alignment Map) file. In the sequence alignment procedure the DNA reads are mapped (in-pair) to the human genome reference sequence by similarity between the sequences. Lastly, loimpute-v0.1.0, Gencove’s imputation procedure, is used over the aligned reads. Imputation is done for the autosomal chromosomes (1-22) using The 1000 Genomes phase 3 haplotypes (1KGP3) as a reference panel, resulting in a VCF (Variant Call Format) file. The VCF contains the most likely genotype (GT) at 37,559,140 positions and their corresponding genotype probabilities (GP). Finally, Gencove employs quality control procedures to exclude samples with poor sequencing and genotyping results.</p>
</section>
</section>
<section id="data-availability" class="level3">
<h3 class="anchored" data-anchor-id="data-availability">Data availability</h3>
<!-- for the example notebooks -->
<p>The information is stored in a number of statistics parquet files:<br> - <code>human_genetics.parquet</code>: sample metadata, including QC statistics, paths to PLINK variant files (raw and post-QC), and principal components (PCs).<br> - <code>variants_qc.parquet</code>: variant QC statistics.<br> - <code>relationship/relationship_ibs.txt</code>: IBS calculated by PLINK for pairs of participants.<br> - <code>relationship/relationship_king.tsv</code>: King kinship coefficients for pairs of participants.<br> - <code>pca/pca.parquet</code>: a PLINK file containing principal components.<br> - <code>pca/pca_loadings.tsv</code>: a PLINK file containing principal component loadings calculated.<br></p>
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">graph LR;
    D[DNA extraction] --&gt;|sequencing| F(FASTQ) --&gt;|alignment| B(BAM) --&gt;|imputation| V(VCF)
    H[REFERENCE HAPLOTYPES] --&gt;|imputation| V --&gt;|"sample&lt;br&gt;aggregation"| R(BED&lt;br&gt;raw) --&gt;|QC| Q(BED&lt;br&gt;post-qc)
    Q --&gt; PCA(PCA)
    Q --&gt; IBS(PLINK IBS&lt;br&gt;relationships)
    Q --&gt; KING(KING&lt;br&gt;relationships)
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</section>
<section id="summary-of-available-data" class="level3">
<h3 class="anchored" data-anchor-id="summary-of-available-data">Summary of available data</h3>
<!-- for the data browser -->
<section id="raw-dna-data" class="level4">
<h4 class="anchored" data-anchor-id="raw-dna-data">Raw DNA data</h4>
<p>Pheno provides per each sample all of the raw sequencing and genotyping workflow data, including FASTQ, BAM, VCF, and QC files.</p>
<ul>
<li>FASTQ files are the sequencing machine outputs. They are text files that store the DNA sequences and their corresponding quality scores. We perform the sequencing at paired-end mode, meaning that each DNA molecule is sequenced from both sides, therefore there are two FASTQ files (R1 and R2).</li>
<li>Binary Alignment Map (BAM) files are a binary and indexed representation of sequence alignment results. The BAM file stores the position of the DNA reads in the reference genome and the corresponding mapping quality. Indexing is provided in the bam.bai file, and allows to retrieve alignments that overlap a specific location quickly.</li>
<li>Variant Call Format (VCF) file is a text file used for storing genetic variations. VCF consists of a header and a table. The header provides metadata describing the table fields and their values. The FILTER field holds a flag indicating whether the variant has failed or successfully passed filters. The raw genotype data includes all 37M positions regardless of their FILTER flag status. Genotypes are flagged as ‘LOWCONF’ at the FILTER field if their maximum GP is less than 0.9, and otherwise flagged as ‘PASS’ quality check. The following FORMAT fields are provided in the VCF - GT:RC:AC:GP:DS (<a href="https://samtools.github.io/hts-specs/VCFv4.3.pdf">VCF specification</a>). Please note that the data is unphased.</li>
<li>PLINK binary biallelic genotype file - In addition to the single sample VCFs we also provide the raw genotype data of all samples jointly in PLINK bed format. Because of the large number of variants, we split the data to chromosomes (22 PLINK bed/bim/bam files). PLINK bed file is a compacted binary representation of genotypes at bi-allelic variants. It is accompanied by .bim and .fam files, which are the metadata files for variants and samples, respectively (<a href="https://www.cog-genomics.org/plink/1.9/formats#bed">PLINK specification</a>).</li>
<li>Gencove QC file - Gencove performs quality control to exclude samples with poor sequencing and genotyping results. Gencove’s quality control matrices are included in the sample metadata.</li>
</ul>
<p>The raw genotype data includes 34,580,532 SNPs and 2,978,606 short indels, 37,494,604 of the variants are bi-alleleic and 30,990 are multi-allelic. Multi-allelic variants are represented in variant normalization format, meaning each alternate allele is listed in a separate line, in both the VCFs and PLINK.</p>
<p>Notably, we do not provide the raw data of samples who failed Gencove QC or for which we identify a possibility of DNA contamination or of mislabelling (i.e.&nbsp;DNA samples are associated with the wrong individual), as explained in section ‘Sample evaluation’.</p>
</section>
<section id="post-genotyping-qc-data" class="level4">
<h4 class="anchored" data-anchor-id="post-genotyping-qc-data">Post-genotyping QC data</h4>
<ul>
<li>We provide the QC genotyped data in PLINK format as explained above. The post-genotyping QC procedure resulted in genotypes for ~99.98% of total samples, and 5,770,633 variants that passed variant QC (~15.36% of the total variants).</li>
<li>In addition to the genotype data we provide auxiliary tabulated datasets such as : variants and sample QC metrics, principal component projection, and familial relationships estimates.</li>
</ul>
</section>
</section>
<section id="relevant-links" class="level3">
<h3 class="anchored" data-anchor-id="relevant-links">Relevant links</h3>
<ul>
<li><a href="https://pheno-demo-app.vercel.app/folder/14">Pheno Data Browser</a></li>
</ul>
<div id="e4e8d45d" class="cell" data-quarto-private-1="{&quot;key&quot;:&quot;execution&quot;,&quot;value&quot;:{&quot;iopub.execute_input&quot;:&quot;2024-04-02T11:45:48.559058Z&quot;,&quot;iopub.status.busy&quot;:&quot;2024-04-02T11:45:48.558776Z&quot;,&quot;iopub.status.idle&quot;:&quot;2024-04-02T11:45:49.837380Z&quot;,&quot;shell.execute_reply&quot;:&quot;2024-04-02T11:45:49.836123Z&quot;}}" data-execution_count="2">
<div class="code-copy-outer-scaffold"><div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pheno_utils <span class="im">import</span> PhenoLoader</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div id="c888255f" class="cell" data-quarto-private-1="{&quot;key&quot;:&quot;execution&quot;,&quot;value&quot;:{&quot;iopub.execute_input&quot;:&quot;2024-04-02T11:45:49.839785Z&quot;,&quot;iopub.status.busy&quot;:&quot;2024-04-02T11:45:49.839462Z&quot;,&quot;iopub.status.idle&quot;:&quot;2024-04-02T11:45:50.841929Z&quot;,&quot;shell.execute_reply&quot;:&quot;2024-04-02T11:45:50.840701Z&quot;}}" data-execution_count="3">
<div class="code-copy-outer-scaffold"><div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>pl <span class="op">=</span> PhenoLoader(<span class="st">'human_genetics'</span>)</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>pl</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<div class="cell-output cell-output-display" data-execution_count="3">
<pre><code>PhenoLoader for human_genetics with
189 fields
3 tables: ['human_genetics', 'pca', 'age_sex']</code></pre>
</div>
</div>
</section>
<section id="data-dictionary" class="level1">
<h1>Data dictionary</h1>
<div id="7f82b1ad" class="cell" data-quarto-private-1="{&quot;key&quot;:&quot;execution&quot;,&quot;value&quot;:{&quot;iopub.execute_input&quot;:&quot;2024-04-02T11:45:50.844207Z&quot;,&quot;iopub.status.busy&quot;:&quot;2024-04-02T11:45:50.843992Z&quot;,&quot;iopub.status.idle&quot;:&quot;2024-04-02T11:45:50.883726Z&quot;,&quot;shell.execute_reply&quot;:&quot;2024-04-02T11:45:50.883264Z&quot;}}" data-execution_count="4">
<div class="code-copy-outer-scaffold"><div class="sourceCode cell-code" id="cb4"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>pl.<span class="bu">dict</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<div class="cell-output cell-output-display" data-execution_count="4">
<div>


<table class="dataframe caption-top table table-sm table-striped small" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">folder_id</th>
<th data-quarto-table-cell-role="th">feature_set</th>
<th data-quarto-table-cell-role="th">field_string</th>
<th data-quarto-table-cell-role="th">description_string</th>
<th data-quarto-table-cell-role="th">field_type</th>
<th data-quarto-table-cell-role="th">bulk_dictionary</th>
<th data-quarto-table-cell-role="th">bulk_file_extension</th>
<th data-quarto-table-cell-role="th">relative_location</th>
<th data-quarto-table-cell-role="th">data_coding</th>
<th data-quarto-table-cell-role="th">stability</th>
<th data-quarto-table-cell-role="th">units</th>
<th data-quarto-table-cell-role="th">sampling_rate</th>
<th data-quarto-table-cell-role="th">strata</th>
<th data-quarto-table-cell-role="th">sexed</th>
<th data-quarto-table-cell-role="th">array</th>
<th data-quarto-table-cell-role="th">completed</th>
<th data-quarto-table-cell-role="th">debut</th>
<th data-quarto-table-cell-role="th">transformation</th>
<th data-quarto-table-cell-role="th">list_of_tags</th>
<th data-quarto-table-cell-role="th">pandas_dtype</th>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">tabular_field_name</th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<th data-quarto-table-cell-role="th">collection_date</th>
<td>14</td>
<td>human_genetics</td>
<td>Collection date</td>
<td>The date of downloading Gencove results from t...</td>
<td>Date</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Time</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>datetime64[ns]</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">version</th>
<td>14</td>
<td>human_genetics</td>
<td>Gencove version</td>
<td>Gencove API version 1 or 2</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">gencove_fastq_r1</th>
<td>14</td>
<td>fastq_bam</td>
<td>R1_FASTQ</td>
<td>Per sample FASTQ file, a text file that contai...</td>
<td>Fastq file (individual)</td>
<td>NaN</td>
<td>R1.fastq.gz</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>string</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">gencove_fastq_r2</th>
<td>14</td>
<td>fastq_bam</td>
<td>R2_FASTQ</td>
<td>Per sample FASTQ file, a text file that contai...</td>
<td>Fastq file (individual)</td>
<td>NaN</td>
<td>R2.fastq.gz</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>string</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">gencove_bam</th>
<td>14</td>
<td>fastq_bam</td>
<td>BAM</td>
<td>Per sample Binary Alignment Map (BAM) file for...</td>
<td>Bulk file (individual)</td>
<td>NaN</td>
<td>bam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>string</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">gencove_bai</th>
<td>14</td>
<td>fastq_bam</td>
<td>BAM indices</td>
<td>Per sample indices for BAM file</td>
<td>Bulk file (individual)</td>
<td>NaN</td>
<td>bam.bai</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>string</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">gencove_vcf</th>
<td>14</td>
<td>fastq_bam</td>
<td>Imputation Variant Call Format (VCF)</td>
<td>Per sample VCF, a text file storing imputed ge...</td>
<td>Bulk file (individual)</td>
<td>NaN</td>
<td>vcf.gz</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>string</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_bases</th>
<td>14</td>
<td>human_genetics</td>
<td>Bases sequenced</td>
<td>number of total bases sequenced. from genecov ...</td>
<td>Integer</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Count</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>int</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_bases_dedup</th>
<td>14</td>
<td>human_genetics</td>
<td>Deduplicated bases</td>
<td>number of deduplicated bases. from genecov qc ...</td>
<td>Integer</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Count</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>int</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">gencove_qc_bases_dedup_mapped</th>
<td>14</td>
<td>human_genetics</td>
<td>Deduplicated bases aligned</td>
<td>number of deduplicated bases that have aligned...</td>
<td>Integer</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Count</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>int</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_effective_coverage</th>
<td>14</td>
<td>human_genetics</td>
<td>Effective coverage</td>
<td>effective coverage. from genecov qc file</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Precent</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_fraction_contamination</th>
<td>14</td>
<td>human_genetics</td>
<td>DNA contamination</td>
<td>contamination by DNA from another sample of th...</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Precent</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_snps</th>
<td>14</td>
<td>human_genetics</td>
<td>SNPs covered</td>
<td>number of variants in reference panel that are...</td>
<td>Integer</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Count</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>int</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_format_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed FASTQ format validity</td>
<td>Indicates proper formatting of the input FASTQ...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_r1_eq_r2_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed paired-end count</td>
<td>Indicates number of bases in R1 file equal to ...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_r1_r2_ids_match_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed paired-end match</td>
<td>Indicates R1 read identifiers match R2 read id...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_bases_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed Bases sequenced</td>
<td>Indicates number of total bases sequenced grea...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_bases_dedup_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed Deduplicated bases</td>
<td>Indicates number of deduplicated bases greater...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_bases_dedup_mapped_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed Deduplicated bases aligned</td>
<td>Indicates number of deduplicated bases aligned...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_effective_coverage_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed Effective coverage</td>
<td>Indicates effective coverage greater than 0. f...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genecov_qc_fraction_contamination_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed DNA contamination</td>
<td>Indicates contamination by DNA less than 0.06....</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">genecov_qc_snps_passed</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed SNPs covered</td>
<td>Indicates number of variants covered by at lea...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Auxiliary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>object</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">heterozygosity_proportion</th>
<td>14</td>
<td>human_genetics</td>
<td>Heterozygosity proportion</td>
<td>Estimated heterozygosity proportion across a s...</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Precent</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pass_het_check</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed Heterozygosity proportion</td>
<td>Indicates heterozygosity proportion is less th...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Supporting</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>bool</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">sample_qc_gp_pass_rate</th>
<td>14</td>
<td>human_genetics</td>
<td>Genotype Probabilities pass rate</td>
<td>Proportion of genotypes with maximum GP greate...</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Precent</td>
<td>NaN</td>
<td>Supporting</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">sample_qc_gp_pass_check</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed Genotype Probabilities</td>
<td>Indicates proportion of GP greater than 0.9 is...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Supporting</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>bool</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">genetic_sex</th>
<td>14</td>
<td>human_genetics</td>
<td>DNA sex</td>
<td>Sex as determined by the percentage of read-al...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>int</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_x_proportion</th>
<td>14</td>
<td>human_genetics</td>
<td>Proportion of alignment to chromosome X</td>
<td>percentage of reads aligned to chromosome X</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Precent</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_y_proportion</th>
<td>14</td>
<td>human_genetics</td>
<td>Proportion of alignment to chromosome Y</td>
<td>percentage of reads aligned to chromosome Y</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Precent</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">sample_qc_gender_match_check</th>
<td>14</td>
<td>human_genetics</td>
<td>Passed gender check</td>
<td>Indicates inferred_gender and submitted_gender...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Supporting</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>bool</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">variants_qc_parquet</th>
<td>14</td>
<td>variants_qc</td>
<td>Variants QC report</td>
<td>QC report for each SNP</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>parquet</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Supporting</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">relatives</th>
<td>14</td>
<td>relationship</td>
<td>Relatives based on IBD estimation</td>
<td>List of second and first degree relatives acco...</td>
<td>Text</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>string</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">relationship_king_tsv</th>
<td>14</td>
<td>relationship</td>
<td>KING kingship file</td>
<td>KING kinship coefficient matrix for pairs of p...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>tsv</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">relationship_ibs_txt</th>
<td>14</td>
<td>relationship</td>
<td>PLINK IBS file</td>
<td>PLINK IBS coefficient matrix for paris of part...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>txt</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_1_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr1</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_1_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr1</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_1_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr1</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_2_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr2</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_2_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr2</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_2_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr2</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_3_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr3</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_3_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr3</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_3_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr3</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_4_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr4</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_4_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr4</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_4_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr4</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_5_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr5</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_5_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr5</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_5_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr5</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_6_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr6</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_6_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr6</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_6_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr6</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_7_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr7</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_7_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr7</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_7_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr7</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_8_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr8</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_8_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr8</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_8_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr8</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_9_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for chr9</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_9_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr9</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_9_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for chr9</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_10_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_10_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr10</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_10_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_11_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_11_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr11</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_11_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_12_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_12_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr12</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_12_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_13_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_13_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr13</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_13_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_14_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_14_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr14</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_14_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_15_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_15_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr15</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_15_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_16_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_16_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr16</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_16_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_17_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_17_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr17</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_17_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_18_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_18_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr18</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_18_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_19_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_19_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr19</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_19_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_20_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_20_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr20</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_20_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_21_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_21_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr21</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_21_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_22_raw_bed</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BED file for the Raw genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_22_raw_fam</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK FAM file accompanying Raw .bed for chr22</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_22_raw_bim</th>
<td>14</td>
<td>genotypes_raw</td>
<td>PLINK BIM file for the Raw genotype data for c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_1_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr1</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_1_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr1</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_1_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_2_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr2</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_2_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr2</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_2_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_3_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr3</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_3_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr3</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_3_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_4_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr4</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_4_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr4</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_4_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_5_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr5</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_5_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr5</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_5_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_6_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr6</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_6_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr6</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_6_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_7_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr7</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_7_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr7</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_7_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_8_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr8</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_8_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr8</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_8_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_9_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for chr9</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_9_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr9</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_9_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_10_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_10_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr10</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_10_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_11_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_11_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr11</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_11_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_12_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_12_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr12</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_12_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_13_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_13_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr13</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_13_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_14_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_14_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr14</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_14_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_15_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_15_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr15</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_15_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_16_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_16_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr16</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_16_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_17_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_17_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr17</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_17_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_18_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_18_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr18</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_18_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_19_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_19_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr19</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_19_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_20_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_20_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr20</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_20_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_21_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_21_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr21</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_21_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_22_postqc_bed</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BED file for post QC genotype data for c...</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bed</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">chr_22_postqc_fam</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK FAM file accompanying QC .bed for chr22</td>
<td>PLINK sample information file accompanying a ....</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>fam</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">chr_22_postqc_bim</th>
<td>14</td>
<td>genotypes_postqc</td>
<td>PLINK BIM file for the post QC genotype data f...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>bim</td>
<td>human_genetics/human_genetics.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc1</th>
<td>14</td>
<td>pca</td>
<td>Principal component 1</td>
<td>Score for projected principal component 1</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc2</th>
<td>14</td>
<td>pca</td>
<td>Principal component 2</td>
<td>Score for projected principal component 2</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc3</th>
<td>14</td>
<td>pca</td>
<td>Principal component 3</td>
<td>Score for projected principal component 3</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc4</th>
<td>14</td>
<td>pca</td>
<td>Principal component 4</td>
<td>Score for projected principal component 4</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc5</th>
<td>14</td>
<td>pca</td>
<td>Principal component 5</td>
<td>Score for projected principal component 5</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc6</th>
<td>14</td>
<td>pca</td>
<td>Principal component 6</td>
<td>Score for projected principal component 6</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc7</th>
<td>14</td>
<td>pca</td>
<td>Principal component 7</td>
<td>Score for projected principal component 7</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc8</th>
<td>14</td>
<td>pca</td>
<td>Principal component 8</td>
<td>Score for projected principal component 8</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc9</th>
<td>14</td>
<td>pca</td>
<td>Principal component 9</td>
<td>Score for projected principal component 9</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc10</th>
<td>14</td>
<td>pca</td>
<td>Principal component 10</td>
<td>Score for projected principal component 10</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc11</th>
<td>14</td>
<td>pca</td>
<td>Principal component 11</td>
<td>Score for projected principal component 11</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc12</th>
<td>14</td>
<td>pca</td>
<td>Principal component 12</td>
<td>Score for projected principal component 12</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc13</th>
<td>14</td>
<td>pca</td>
<td>Principal component 13</td>
<td>Score for projected principal component 13</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc14</th>
<td>14</td>
<td>pca</td>
<td>Principal component 14</td>
<td>Score for projected principal component 14</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc15</th>
<td>14</td>
<td>pca</td>
<td>Principal component 15</td>
<td>Score for projected principal component 15</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc16</th>
<td>14</td>
<td>pca</td>
<td>Principal component 16</td>
<td>Score for projected principal component 16</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc17</th>
<td>14</td>
<td>pca</td>
<td>Principal component 17</td>
<td>Score for projected principal component 17</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc18</th>
<td>14</td>
<td>pca</td>
<td>Principal component 18</td>
<td>Score for projected principal component 18</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">pc19</th>
<td>14</td>
<td>pca</td>
<td>Principal component 19</td>
<td>Score for projected principal component 19</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pc20</th>
<td>14</td>
<td>pca</td>
<td>Principal component 20</td>
<td>Score for projected principal component 20</td>
<td>Continuous</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>NaN</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>float</td>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">used_in_pca_calculation</th>
<td>14</td>
<td>pca</td>
<td>Included in principal component analysis</td>
<td>Indicates samples was in the input for princip...</td>
<td>Categorical (single)</td>
<td>NaN</td>
<td>NaN</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Boolean</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>bool</td>
</tr>
<tr class="even">
<th data-quarto-table-cell-role="th">pca_loadings_tsv</th>
<td>14</td>
<td>pca</td>
<td>Principal component loading file</td>
<td>Principal component loading file</td>
<td>Bulk file (group)</td>
<td>NaN</td>
<td>tsv</td>
<td>human_genetics/pca/pca.parquet</td>
<td>NaN</td>
<td>Accruing</td>
<td>Text</td>
<td>NaN</td>
<td>Primary</td>
<td>Both sexes</td>
<td>Single</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Human Genetics</td>
<td>category</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>